src/main/java/com/exasol/adapter/installer/Installer.java

```diff
@@ -232,32 +222,32 @@ public InstallerBuilder jdbcDriver(final JdbcDriver jdbcDriver) {
         }
 
         public InstallerBuilder exaUsername(final String exaUsername) {
-            this.exaUsername = InputString.validate(exaUsername);
+            this.exaUsername = exaUsername;
             return this;
         }
 
         public InstallerBuilder exaPassword(final String exaPassword) {
-            this.exaPassword = InputString.validate(exaPassword);
+            this.exaPassword = exaPassword;
             return this;
         }
 
         public InstallerBuilder exaBucketWritePassword(final String exaBucketWritePassword) {
-            this.exaBucketWritePassword = InputString.validate(exaBucketWritePassword);
+            this.exaBucketWritePassword = exaBucketWritePassword;
             return this;
         }
 
         public InstallerBuilder sourceUsername(final String sourceUsername) {
-            this.sourceUsername = InputString.validate(sourceUsername);
+            this.sourceUsername = UserInputValidator.validateLiteralString(sourceUsername);
             return this;
         }
 
         public InstallerBuilder sourcePassword(final String sourcePassword) {
-            this.sourcePassword = InputString.validate(sourcePassword);
+            this.sourcePassword = UserInputValidator.validateLiteralString(sourcePassword);
             return this;
         }
 
         public InstallerBuilder exaIp(final String exaIp) {
-            this.exaIp = InputString.validate(exaIp);
+            this.exaIp = exaIp;
             return this;
```

Comment:
Do we need validation for this? Previously it was also validated. Same for username, password and bucket password. Maybe username still need validation.

Reply:
I added validation to the values that are used in the SQL statementss. ExaIP, username and password are only used to create a connection to Exasol via Java code (with JDBC driver). So I think we don't really need to validate them. Bucket password is also used only in code